modules/oadp-ibm-z-test-support.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+:_content-type: CONCEPT
+[id="oadp-ibm-z-test-support_{context}"]
+= OADP testing and support for target backup locations using IBM Z
+
+IBM Z ("bare-metal") running on OCP 4.11 and OADP 1.1.0 was tested successfully aganst IBM and AWS S3 backup location targets. Although this combination was not tested against non-AWS S3 backup location targets, such as GCP and Azure, Red Hat supports running IBM Z with OCP 4.11 and OADP 1.1.0 againt them.
```

Comment:
>Although this combination was not tested against non-AWS S3 backup location targets, such as GCP and Azure,
Would better say: `Although this combination was tested only against AWS S3 for backup location`, without mention which not, as it was not tested also for MCG, Ceph, and many other S3 compatible storages out there...

Reply:
@mperetzred GCP and Azure are given as examples, so the sentence is OK -- but I have changed the text along the lines you suggested.